components/common/Text/TextBody.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextProps } from './TextProps';
+
+const Text = styled.span`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 22px;
+    color: ${({ theme, color }) => color ? color : theme.text.defaultColor};
+`;
+
+const TextBody = (props: TextProps) => (
```

Comment:
The component is useless. You can remove it and change `const Text = styled.span` to `const TextBody = styled<TextProps>`.

Reply:
replaced textBody component with Text component